doc/source/visualizing/callbacks.rst

```diff
@@ -22,7 +22,7 @@ of available callbacks.  For example:
 
 .. code-block:: python
 
-   slc = SlicePlot(ds, 0, "density")
+   slc = SlicePlot(ds, "x", "density")
```

Comment:
don't we want to do `("gas", "density")` here too?

Reply:
Probably! I'll take a crack at this and the rest.

Reply:
Awesome!

Reply:
Resolving!